internals/feature_links.py

```diff
@@ -80,6 +81,8 @@ def _get_index_link(link: Link, fe: FeatureEntry, should_parse_new_link: bool =
   else:
     if should_parse_new_link:
       link.parse()
+      if link.is_error:
```

Comment:
If you don't index links that have errors, then how will you generate notification emails about broken links?

Reply:
During the meeting we agreed that we would only generate notifications for links that were valid before and turned broken later, not for links that were never valid.